app/views/attendances/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_for(@attendance) do |f| %>
+  <% if @attendance.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@attendance.errors.count, "error") %> prohibited this attendance from being saved:</h2>
+
+      <ul>
+      <% @attendance.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :date %><br>
+    <%= f.text_field :date %>
```

Comment:
You should check out the `date_select` method: http://apidock.com/rails/ActionView/Helpers/DateHelper/date_select

You can use that instead of `text_field` and it will give you nice year, month, day dropdowns.